Directory.Build.props

```diff
@@ -1,5 +1,5 @@
 <Project>
 	<PropertyGroup>
-        <Version>6.1.0</Version>
+        <Version>6.1.1</Version>
```

Comment:
Not your fault but this will be out-of-date and cause a merge conflict, as other PRs have moved on. I would suggest removing your change to this file and pick up the version change coming in from other PRs.